docs/userguide/forecasting_overview.md

```diff
@@ -40,6 +40,54 @@ Furthermore, we define the following types of time series consumed by the models
 * **Target series:** the series that we are interested in forecasting.
 * **Covariate series:** some other series that we are not interested in forecasting, but that can provide valuable inputs to the forecasting model.
 
+## Reproducibility
+
+If the user wishes to save a particular model and use it elsewhere or at a later point in time, darts leverages pickle to achieve that.
```

Comment:
[nitpicking] You shouldn't talk about the user in third person. Either don't address the reader, or say "you".

Reply:
Noted and fixed.